### ISSUE TITLE
Unable to start activity ComponentInfo{org.thoughtcrime.securesms/org.thoughtcrime.securesms.conversation.ConversationActivity}

### ISSUE THREAD

**Issue by kaovilai** [signalapp/Signal-Android#11642]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.
Similar to https://github.com/signalapp/Signal-Android/issues/10589 but not resolved via suggested solutions. Signal have all permissions granted.

### Steps to reproduce
- Launch Signal
- Open existing conversations

**Actual result:** Signal Crash
**Expected result:** Conversation open

### Device info

**Device:** OnePlus 7 Pro
**Android version:** 11
**Signal version:** 5.23.8

### Link to debug log

[logcat](https://gist.github.com/kaovilai/b9821d36842dbcc4999f9842c8d1c131#file-signal-logcat)
[getprop](https://gist.github.com/kaovilai/b9821d36842dbcc4999f9842c8d1c131#file-getprop)

Including this logcat snippet for similar issue search visibility
```
09-22 22:51:39.796 30766 30766 E AndroidRuntime: 	at org.thoughtcrime.securesms.util.SmsCharacterCalculator.calculateCharacters(SmsCharacterCalculator.java:35)

```

**Comment by kaovilai** [comment]
After plugging phone in to charge or simply a few 10 minutes or so passing, the app no longer crashes.

I was able to disable permissions previously mentioned as required and still not cause another crash upon opening conversation. There have been no app updates to resolve this. Feel free to close this but leaving open for now so you are aware the code could still crash today.

**Comment by cody-signal** [comment]
This is fixed in 5.31 (beta coming soon)
